src/EnergyPlus/ScheduleManager.cc

```diff
@@ -562,7 +562,7 @@ namespace ScheduleManager {
                                 CSVAllColumnNameAndValues[colCnt - 1] = timestepColumnValues;
                             }
                         } else {
-                            columnValue = std::stod(subString, nullptr);
+                            columnValue = UtilityRoutines::ProcessNumber(subString, errFlag);
```

Comment:
It is worth noting that this version is faster on my local machine while having no diffs. 
Ran in 5.74 seconds. 
![image](https://user-images.githubusercontent.com/45446967/121533144-3af3f300-c9bd-11eb-9c08-3b74a70c7c8b.png)